src/script/lua_api/l_object.cpp

```diff
@@ -1434,14 +1434,17 @@ int ObjectRef::l_set_physics_override(lua_State *L)
 	if (playersao == nullptr)
 		return 0;
 
+	RemotePlayer *player = playersao->getPlayer();
+	auto &phys = player->physics_override;
```

Comment:
shouldn't it be part of `PlayerSAO` if it's almost always only used there?

Reply:
Physics- and input related code is contained in the `Player` class, which `RemotePlayer` and `LocalPlayer` inherits. PlayerSAO however is more of a glue between Player <---> ServerActiveObject, thus I think the code belongs better to `Player` so that it can be shared by client and server using the existing inheritance hierarchy.